hedera-node/hedera-network-admin-service-impl/src/test/java/com/hedera/node/app/service/networkadmin/impl/test/handlers/NetworkAdminHandlerTestBase.java

```diff
@@ -207,7 +207,7 @@ protected void refreshRecordCache() {
         lenient().when(wsa.getHederaState()).thenReturn(state);
         lenient().when(props.getConfiguration()).thenReturn(versionedConfig);
         lenient().when(versionedConfig.getConfigData(HederaConfig.class)).thenReturn(hederaConfig);
-        lenient().when(hederaConfig.transactionMaxValidDuration()).thenReturn(180L);
+        lenient().when(hederaConfig.transactionMaxValidDuration()).thenReturn(123456789999L);
```

Comment:
The typical configured value is 180 (3 min). Why does this value have to be increased so dramatically in order to pass?

Reply:
It didn't.  Lev just chose a value that could never be exceeded as the intention for this particular test value is to not expire events.
It does have to be fairly excessive as there is a bit of "current time" impact that needed to be accommodated, but setting the value the same as the mock consensus time is probably fine.